github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -28,13 +28,19 @@ presubmits:
       preset-bazel-cache: "true"
     spec:
       containers:
-      - image: quay.io/kubevirtci/bootstrap:v20230801-94954c0
+      - image: quay.io/kubevirtci/golang:v20230626-e1b7af2
         command:
-          - "/bin/bash"
-          - "-c"
-          - |
-            make gazelle-update-repos
-            bazelisk test //robots/...
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -ce
+        - |
+          make -C robots/cmd/test-label-analyzer install-ginkgo
```

Comment:
should we have `install-ginko` available in the robots dir?

Reply:
Good point. Maybe there should even be a dedicated image `images/test-label-analyzer` that contains the `test-label-analyzer` binary and `ginkgo` binary.

WDYT?